### PR TITLE
update the argument for `export_as_pdf` in table_with_settings

### LINKED ISSUE #208: Address R CMD Check Error
Need to fix findings from integration test:

```r
* checking tests ...
  Running ‘testthat.R’
 ERROR
Running the tests in ‘tests/testthat.R’ failed.
Complete output:
  > pkg_name <- "teal.widgets"
  > library(pkg_name, character.only = TRUE)
  > testthat::test_check(pkg_name)
  Error in pag_indices_inner(pagdf = mf_rinfo(mpf), rlpp = calc_rlpp(pg_size_spec,  : 
    Lines of repeated context (plus header materials) larger than specified lines per page
  [ FAIL 3 | WARN 0 | SKIP 0 | PASS 202 ]
  
  ══ Failed tests ════════════════════════════════════════════════════════════════
  ── Error ('test-table_with_settings.R:65:9'): type_download_srv_table: downloading different output types, custom name ──
  Error in `rtables::export_as_pdf(tt = table_reactive(), file = file, paginate = input$pagination_switch, 
      lpp = if (input$pagination_switch) as.numeric(input$lpp))`: unused argument (tt = table_reactive())
  Backtrace:
       ▆
    1. ├─shiny::testServer(...) at test-table_with_settings.R:57:3
    2. │ ├─shiny:::withMockContext(...)
    3. │ │ ├─shiny::isolate(...)
    4. │ │ │ ├─shiny::..stacktraceoff..(...)
    5. │ │ │ └─ctx$run(...)
    6. │ │ │   ├─promises::with_promise_domain(...)
    7. │ │ │   │ └─domain$wrapSync(expr)
    8. │ │ │   ├─shiny::withReactiveDomain(...)
    9. │ │ │   │ └─promises::with_promise_domain(...)
   10. │ │ │   │   └─domain$wrapSync(expr)
   11. │ │ │   │     └─base::force(expr)
   12. │ │ │   └─env$runWith(self, func)
   13. │ │ │     └─shiny (local) contextFunc()
   14. │ │ │       └─shiny::..stacktraceon..(expr)
   15. │ │ ├─shiny::withReactiveDomain(...)
   16. │ │ │ └─promises::with_promise_domain(...)
   17. │ │ │   └─domain$wrapSync(expr)
   18. │ │ │     └─base::force(expr)
   19. │ │ └─withr::with_options(...)
   20. │ │   └─base::force(code)
   21. │ └─rlang::eval_tidy(quosure, mask, rlang::caller_env())
   22. ├─testthat::expect_true(file.exists(output$data_download)) at test-table_with_settings.R:65:9
   23. │ └─testthat::quasi_label(enquo(object), label, arg = "object")
   24. │   └─rlang::eval_bare(expr, quo_get_env(quo))
   25. ├─base::file.exists(output$data_download)
   26. ├─output$data_download
   27. └─shiny:::`$.shinyoutput`(output, data_download)
   28.   └─.subset2(x, "impl")$getOutput(name)
   29.     └─private$renderFile(self$ns(name), download)
   30.       └─download$content(file)
  ── Error ('test-table_with_settings.R:83:9'): type_download_srv_table: downloading different output types, no name ──
  Error in `rtables::export_as_pdf(tt = table_reactive(), file = file, paginate = input$pagination_switch, 
      lpp = if (input$pagination_switch) as.numeric(input$lpp))`: unused argument (tt = table_reactive())
  Backtrace:
       ▆
    1. ├─shiny::testServer(...) at test-table_with_settings.R:76:3
    2. │ ├─shiny:::withMockContext(...)
    3. │ │ ├─shiny::isolate(...)
    4. │ │ │ ├─shiny::..stacktraceoff..(...)
    5. │ │ │ └─ctx$run(...)
    6. │ │ │   ├─promises::with_promise_domain(...)
    7. │ │ │   │ └─domain$wrapSync(expr)
    8. │ │ │   ├─shiny::withReactiveDomain(...)
    9. │ │ │   │ └─promises::with_promise_domain(...)
   10. │ │ │   │   └─domain$wrapSync(expr)
   11. │ │ │   │     └─base::force(expr)
   12. │ │ │   └─env$runWith(self, func)
   13. │ │ │     └─shiny (local) contextFunc()
   14. │ │ │       └─shiny::..stacktraceon..(expr)
   15. │ │ ├─shiny::withReactiveDomain(...)
   16. │ │ │ └─promises::with_promise_domain(...)
   17. │ │ │   └─domain$wrapSync(expr)
   18. │ │ │     └─base::force(expr)
   19. │ │ └─withr::with_options(...)
   20. │ │   └─base::force(code)
   21. │ └─rlang::eval_tidy(quosure, mask, rlang::caller_env())
   22. ├─testthat::expect_true(file.exists(output$data_download)) at test-table_with_settings.R:83:9
   23. │ └─testthat::quasi_label(enquo(object), label, arg = "object")
   24. │   └─rlang::eval_bare(expr, quo_get_env(quo))
   25. ├─base::file.exists(output$data_download)
   26. ├─output$data_download
   27. └─shiny:::`$.shinyoutput`(output, data_download)
   28.   └─.subset2(x, "impl")$getOutput(name)
   29.     └─private$renderFile(self$ns(name), download)
   30.       └─download$content(file)
  ── Error ('test-table_with_settings.R:102:9'): type_download_srv_table: downloading different output types, pagination ──
  Error in `rtables::export_as_pdf(tt = table_reactive(), file = file, paginate = input$pagination_switch, 
      lpp = if (input$pagination_switch) as.numeric(input$lpp))`: unused argument (tt = table_reactive())
  Backtrace:
       ▆
    1. ├─shiny::testServer(...) at test-table_with_settings.R:93:3
    2. │ ├─shiny:::withMockContext(...)
    3. │ │ ├─shiny::isolate(...)
    4. │ │ │ ├─shiny::..stacktraceoff..(...)
    5. │ │ │ └─ctx$run(...)
    6. │ │ │   ├─promises::with_promise_domain(...)
    7. │ │ │   │ └─domain$wrapSync(expr)
    8. │ │ │   ├─shiny::withReactiveDomain(...)
    9. │ │ │   │ └─promises::with_promise_domain(...)
   10. │ │ │   │   └─domain$wrapSync(expr)
   11. │ │ │   │     └─base::force(expr)
   12. │ │ │   └─env$runWith(self, func)
   13. │ │ │     └─shiny (local) contextFunc()
   14. │ │ │       └─shiny::..stacktraceon..(expr)
   15. │ │ ├─shiny::withReactiveDomain(...)
   16. │ │ │ └─promises::with_promise_domain(...)
   17. │ │ │   └─domain$wrapSync(expr)
   18. │ │ │     └─base::force(expr)
   19. │ │ └─withr::with_options(...)
   20. │ │   └─base::force(code)
   21. │ └─rlang::eval_tidy(quosure, mask, rlang::caller_env())
   22. ├─testthat::expect_true(file.exists(output$data_download)) at test-table_with_settings.R:102:9
   23. │ └─testthat::quasi_label(enquo(object), label, arg = "object")
   24. │   └─rlang::eval_bare(expr, quo_get_env(quo))
   25. ├─base::file.exists(output$data_download)
   26. ├─output$data_download
   27. └─shiny:::`$.shinyoutput`(output, data_download)
   28.   └─.subset2(x, "impl")$getOutput(name)
   29.     └─private$renderFile(self$ns(name), download)
   30.       └─download$content(file)
  
  [ FAIL 3 | WARN 0 | SKIP 0 | PASS 202 ]
  Error: Test failures
  Execution halted
```

### PR DESCRIPTION
**Pull request by donyunardi**
Fixes #208

### CODE REVIEW

File: R/table_with_settings.R [hunk 1]
```diff
@@ -72,7 +72,7 @@ table_with_settings_ui <- function(id, ...) {
 #'   }
 #' )
 #' if (interactive()) {
-#'   shinyApp(app$ui, app$server)
+#'   app
```
Comment (chlebowa) [inline R/table_with_settings.R:75]: Why this change? We just changed `runApp` to `shinyApp` throughout.

Comment (vedhav) [reply R/table_with_settings.R:75]: It was my bad. I overlooked it while making that change. `app` is teal app! There were 3 or 4 such examples. I missed this one.

Comment (chlebowa) [reply R/table_with_settings.R:75]: Not a `teal` app but an app it is. I thought it was agreed to use `shinyApp` in all examples. Perhaps I missed something.

Comment (chlebowa) [reply R/table_with_settings.R:75]: OK, now I'm confused. If `app` is an app, the example should conclude with running it, should it not? If the point of the example is to _print_ the app, hten the `if (interactive())` is not necessary. Please explain.

Comment (chlebowa) [reply R/table_with_settings.R:75]: Oooh, I forgot `init` doesn't actually return an app object :sweat_smile:  Sorry about that, all good.

**Review by pawelru — CHANGES_REQUESTED** [review]
This PR should include change in dependency version.

**Comment by donyunardi** [comment]
Ah, thanks for bumping rtables, forgot to do that.
Merging.
